api/config/config.js

```diff
@@ -1,11 +1,12 @@
-const env = process.env.NODE_ENV || 'development';
+require('dotenv').config();
```

Comment:
You can add this line in the startup file `require('dotenv').config();` just to configure the dotenv and get rid of the whole file because it is not needed. Whenever you want for example to use the port just write process.env.PORT and it will automatically get the data from your .env file, which you also should create.